tests/OSModulesTest.php

```diff
@@ -98,6 +98,10 @@ public function testOS($os, $variant, $modules): void
         $this->requireSnmpsim();  // require snmpsim for tests
         // stub out Eventlog::log and Fping->ping, we don't need to store them for these tests
         $this->stubClasses();
+        // Order modules following both disovery and poller
```

Comment:
I don't think this is the correct place for this. I'll try to send a PR just to fix this issue.

Reply:
Not 100% sure that the your PR code is run in this case, still unclear for me. But I removed the poller part in `tests/OSModulesTest.php` and we'll see how it behaves